cli/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": [
+    "oclif",
+    "oclif-typescript"
```

Comment:
would be nice to add in the future eslint rules to check for `;` indent and spaces